client/src/components/new-request/submit-request/reserve-submit.vue

```diff
@@ -2,20 +2,15 @@
   <v-tooltip bottom
              content-class="bottom-tooltip"
              transition="fade-transition"
-              :disabled="isContinue">
+             :disabled="isContinue">
     <template v-slot:activator="scope">
```

Comment:
Actually this tooltip doesn't work, but I couldn't fix it quickly so I left it.

Reply:
How so? It works in dev, or do you mean it doesn't work in some other way?

Reply:
Well, when I hover over the Continue button, the tooltip doesn't appear.

![image](https://user-images.githubusercontent.com/10002889/103059728-cdf9b780-455a-11eb-8eb3-1c2a87aeb901.png)

Reply:
This button has multiple implementations, it's confusing for sure. 
The tooltip scenario, where it works and is intended for is during it's implementation in the analyze-pending component i think.
it's disabled by default, ie when 'isContinue'.